client/src/main/java/de/rebleyama/client/RebleyamaClient.java

```diff
@@ -66,22 +78,11 @@ public boolean keyDown(int keycode) {
 
     @Override
     public boolean keyUp(int keycode) {
```

Comment:
Methods that are intentionally left as stubs should be commented as such.

Reply:
Added javadoc comments to them to indicate what they'll be used for